src/ucp/wireup/address.h

```diff
@@ -60,8 +60,7 @@ enum {
                                             ~UCP_ADDRESS_PACK_FLAG_TL_RSC_IDX,
 
     UCP_ADDRESS_PACK_FLAGS_CM_DEFAULT     = UCP_ADDRESS_PACK_FLAG_IFACE_ADDR |
-                                            UCP_ADDRESS_PACK_FLAG_EP_ADDR    |
-                                            UCP_ADDRESS_PACK_FLAG_TL_RSC_IDX,
```

Comment:
where is `UCP_ADDRESS_PACK_FLAG_TL_RSC_IDX` used now?

Reply:
It is part of `UCP_ADDRESS_PACK_FLAGS_ALL`

Reply:
they are sent in WIREUP messages only:
https://github.com/openucx/ucx/blob/9f4f08766db198fb43b212d45dfd3187ce475c93/src/ucp/wireup/wireup.c#L201

Reply:
a, ok